CHANGELOG.md

```diff
@@ -5,9 +5,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/). Versionin
 
 ## 1.157.0
 ### Added
-- Added "Configure Pipeline", which helps you configure a pipeline for Azure deployment
-### Fixed
-- Updated a few vulnerable npm packages (nothing that would affect end users)
+- Added "Configure Pipeline" option in Command Palette( Ctrl + Shift + P ) and for right click File Explorer. This will configure a continuous deployment pipeline to Azure Web App
```

Comment:
Feel free to edit this one: Added "Configure Pipeline" option in Command Palette and for right click File Explorer. This will configure a CI and CD pipeline for deploying web applications to Azure Windows Web App using 'Azure Pipelines'

Reply:
Took some of the suggestion and updated the changeLog